### ISSUE TITLE
FNX2-15113 ⁃ [Bug] Native crash [@ nsNoDataProtocolContentPolicy::ShouldLoad] (mozilla::css::Loader::CheckContentPolicy) - "audio.onet.pl"

### ISSUE THREAD

**Issue by rafalopilowski1** [mozilla-mobile/fenix#12249]
This bug is for crash reports:

* [bp-2c5be5a3-65d8-4c3e-a5ab-974b40200703](https://crash-stats.mozilla.org/report/index/2c5be5a3-65d8-4c3e-a5ab-974b40200703)
* [bp-a8210c32-7d24-424f-8a65-69f7c0200703](https://crash-stats.mozilla.org/report/index/a8210c32-7d24-424f-8a65-69f7c0200703)
* [bp-0ba4ca36-6671-4c90-beb9-f75fe0200703](https://crash-stats.mozilla.org/report/index/0ba4ca36-6671-4c90-beb9-f75fe0200703)
* [bp-c8a41ffb-1169-4907-8d1d-b75070200703](https://crash-stats.mozilla.org/report/index/c8a41ffb-1169-4907-8d1d-b75070200703) (version from 03.07.2020)

Issue at Bugzilla: [https://bugzilla.mozilla.org/show_bug.cgi?id=1650409](https://bugzilla.mozilla.org/show_bug.cgi?id=1650409)

### STR:
1. Open Firefox Nightly on Android
1. Navigate to "[https://audio.onet.pl](https://audio.onet.pl)"

### Expected behaviour:
Website "[https://audio.onet.pl](https://audio.onet.pl)" loads correctly.

### Actual behaviour:
Recoverable native crashes reported to Socorro - website "[https://audio.onet.pl](https://audio.onet.pl)" is unaccessible.

[Screencast of the issue](https://streamable.com/xqzlb7)

### Device information
* Android device: OnePlus 6 (Android 10, Oxygen 10.3.4)
* Fenix versions:

> Nightly 200702 06:03 (Build #21840612)
   > AC: 49.0.20200701130905, 14d09cc4a
   > GV: 79.0a1-20200629094229
   > AS: 61.0.7

> Nightly 200703 08:40 (Build #21850844)
   > AC: 49.0.20200702131412, 1cba216b2
   > GV: 80.0a1-20200701093012
   > AS: 61.0.7

Top 10 frames of crashing thread:

```
0 libxul.so nsNoDataProtocolContentPolicy::ShouldLoad dom/base/nsNoDataProtocolContentPolicy.cpp:44
1 libxul.so nsContentPolicy::CheckPolicy dom/base/nsContentPolicy.cpp:123
2 libxul.so nsContentPolicy::ShouldLoad dom/base/nsContentPolicy.cpp:172
3 libxul.so NS_CheckContentLoadPolicy dom/base/nsContentPolicyUtils.h:226
4 libxul.so mozilla::css::Loader::CheckContentPolicy layout/style/Loader.cpp:898
5 libxul.so mozilla::css::Loader::InternalLoadNonDocumentSheet layout/style/Loader.cpp:2031
6 libxul.so mozilla::css::Loader::LoadSheet layout/style/Loader.cpp:2007
7 libxul.so mozilla::dom::Document::PreloadStyle dom/base/Document.cpp:11577
8 libxul.so mozilla::PreloadService::PreloadStyle uriloader/preload/PreloadService.cpp:198
9 libxul.so mozilla::PreloadService::PreloadOrCoalesce uriloader/preload/PreloadService.cpp:166
```

**Comment by rafalopilowski1** [comment]
Confirmed also for Firefox Nightly 80.0a1 on Windows 10 (Build #20200703094420)

Socorro: [crash report bp-b09b58c0-1fec-4c46-a40e-5968c0200703](https://crash-stats.mozilla.org/report/index/b09b58c0-1fec-4c46-a40e-5968c0200703)

It seems to be the problem with Gecko, not Fenix itself.

**Comment by rafalopilowski1** [comment]
**Verified as fixed on [BZ-1650409](https://bugzilla.mozilla.org/show_bug.cgi?id=1650409)**

Quote from [comment](https://bugzilla.mozilla.org/show_bug.cgi?id=1650409#c9) at Bugzilla:
> [URL](https://audio.onet.pl) mentioned in this issue now loads properly:
>
>  - [Screencast from Fenix Nightly](https://streamable.com/7vqtqn)
